projects/1.18/assets/delightful/delightful/lang/zh_cn.json

```
@@ -0,0 +1,90 @@
+{
+  "item.delightful.cheeseburger": "芝士汉堡",
+  "item.delightful.deluxe_cheeseburger": "豪华芝士汉堡",
+  "item.delightful.marshmallow_stick": "插棍棉花糖",
+  "item.delightful.cooked_marshmallow_stick": "插棍烤棉花糖",
+  "item.delightful.smore": "烤棉花糖饼干",
+  "item.delightful.crab_rangoon": "炸蟹角",
+  "item.delightful.prickly_pear_juice": "仙人掌果汁",
+  "item.delightful.chunkwich": "糖腌厚腐肉片三明治",
+  "item.delightful.chunk_nugget": "糖腌腐肉粒",
+  "item.delightful.jelly_bottle": "果酱瓶",
+  "item.delightful.glow_jelly_bottle": "发光果酱瓶",
+  "item.delightful.nut_butter_bottle": "果仁奶油瓶",
+  "item.delightful.nut_butter_and_jelly_sandwich": "果仁奶油果酱三明治",
+  "item.delightful.ender_nectar": "末影甘露",
+  "item.delightful.azalea_tea": "杜鹃花茶",
+  "item.delightful.lavender_tea": "薰衣草茶",
+  "item.delightful.honey_glazed_walnut": "琥珀核桃",
+  "item.delightful.matcha_latte": "抹茶拿铁",
+  "item.delightful.berry_matcha_latte": "浆果抹茶拿铁",
+  "item.delightful.green_tea_leaf": "绿茶叶",
+  "item.delightful.salmonberries": "大树莓",
+  "item.delightful.salmonberry_pips": "大树莓籽",
+  "item.delightful.matcha": "抹茶",
+  "item.delightful.acorn": "橡子",
+  "item.delightful.animal_fat": "动物脂肪",
+  "item.delightful.animal_oil_bottle": "荤油瓶",
+  "item.delightful.chopped_clover": "切碎的三叶草",
+  "item.delightful.cactus_flesh": "仙人掌肉",
+  "item.delightful.cactus_steak": "仙人掌肉排",
+  "item.delightful.field_salad": "田园沙拉",
+  "item.delightful.rock_candy": "石头糖",
+  "item.delightful.venison_chops": "生鹿排",
+  "item.delightful.cooked_venison_chops": "熟鹿排",
+  "item.delightful.raw_goat": "生山羊肉",
+  "item.delightful.cooked_goat": "熟山羊肉",
+  "item.delightful.bone_knife": "骨刀",
+  "item.delightful.amethyst_knife": "紫水晶刀",
+  "item.delightful.emerald_knife": "绿宝石刀",
+  "item.delightful.copper_knife": "铜刀",
+  "item.delightful.tin_knife": "锡刀",
+  "item.delightful.steel_knife": "钢刀",
+  "item.delightful.enderite_knife": "末影合金刀",
+  "item.delightful.obsidian_infused_enderite_knife": "黑曜石注能末影合金刀",

```

```suggestion
  "item.delightful.enderite_knife": "末影合金刀",
  "item.delightful.obsidian_infused_enderite_knife": "黑曜石灌注末影合金刀",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/projects/1.18/assets/enderite-lieonlion/lolenderite/lang/zh_cn.json